scripts/scr_company_order/scr_company_order.gml

```diff
@@ -163,6 +221,38 @@ function scr_company_order(company) {
 			}		
 		}
 	}
+
+	var squadless_and_squad_spaces = [squadless,empty_squads];
+
+	var squad_builder = [
+		["tactical_squad",role[100][8],5],
```

Comment:
could you remind me what was preventing use from refactoring `role[100][8]` into `roles.tactical_squad`? I remember we discussed this before but I couldn't find the messages anywhere

Reply:
ok, I'll create a separate issue for this